test/InterestRateStrategyBase.t.sol

```diff
@@ -66,7 +66,8 @@ abstract contract InterestRateStrategyBaseTest is Test {
             uint256 actualVariableBorrowRate
         ) = interestStrategy.calculateInterestRates(params);
 
-        assertEq(expectedLiquidityRate,      actualLiquidityRate,      "liquidity rate mismatch");
+        // Small diff from simplifying the stable borrow stuff
+        assertApproxEqRel(expectedLiquidityRate, actualLiquidityRate, 0.0000001e18, "liquidity rate mismatch");
```

Comment:
Stable borrow is just set to 0, which was also a valid state in the Default IRS. Could you please elaborate where is this diff coming from?

Reply:
https://github.com/marsfoundation/aave-v3-core/blob/master/contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol#L236 this function computes the combined variable + stable borrow rate to convert to the liquidity rate. When the stable borrow is disabled the formula simplifies just to be the variable borrow rate instead of the weighted average. Removing the calculation removes some of the rounding errors hence the approximate equivalence.

Reply:
Ok, I see. Makes sense.